src/nomad_simulations/schema_packages/properties/molecular_orbitals.py

```diff
@@ -0,0 +1,138 @@
+import itertools
+from collections.abc import Iterable
+from typing import TYPE_CHECKING, Any, Optional
+
+if TYPE_CHECKING:
+    from nomad.datamodel.datamodel import EntryArchive
+    from structlog.stdlib import BoundLogger
+
+import numpy as np
+from nomad.datamodel.data import ArchiveSection
+from nomad.datamodel.metainfo.basesections.v2 import Entity
+from nomad.metainfo import URL, MEnum, Quantity, SectionProxy, SubSection
+
+from nomad_simulations.schema_packages.physical_property import PhysicalProperty
+from nomad_simulations.schema_packages.variables import Variables
+
+
+class MolecularOrbitals(PhysicalProperty):
```

Comment:
Is there a "main" value for this property? Does it even make sense in this case?

I am wondering in the context of using `value` or some sub-section for different types of the "main" value. I guess we never discussed what to do if the property is only really defined in terms of a set of equally-important metadata